common/test/acceptance/pages/studio/auto_auth.py

```diff
@@ -15,7 +15,7 @@ class AutoAuthPage(PageObject):
     this url will create a user and log them in.
     """
 
-    def __init__(self, browser, username=None, email=None, password=None, staff=None, course_id=None, roles=None):
+    def __init__(self, browser, username=None, email=None, password=None, staff=None, course_id=None, roles=None, no_login=None):
```

Comment:
Note to reviewers: these changes were taken from #6283, which has not yet merged to master.